wiring/inc/spark_wiring_cellular.h

```diff
@@ -74,6 +74,12 @@ class CellularClass : public NetworkClass
 
     CellularSignal RSSI();
 
+    CellularData dataUsage(void);
+
+    CellularData dataUsage(const CellularData &data_set);
```

Comment:
Since the returned CellularData is not globally defined and known to the system API, I figured returning a copy was a safe bet.  It also sounds like there are return value optimizations that are done that keep things fast.  If we return by reference, can you suggest how this might be architected?